src/server/code-parse/language/javascript/dependencies.js

```diff
@@ -67,6 +32,6 @@ const getDependencies = (entryPoint, projectDir, webpackConfigPath) => {
 };
 
 module.exports = {
-  getImports,
+  getImports: (fileCode, itemPath) => jsDependencies.getImports(fileCode, itemPath, astParseConfig, getNodeLines),
```

Comment:
duplicating config, but keeping parser as common utils, fine for you @Bogdan-Lyashenko ?